server/base/src/main/java/org/apache/accumulo/server/mem/LowMemoryDetector.java

```diff
@@ -36,20 +36,21 @@
 
 public class LowMemoryDetector {
 
+  private static final Logger LOG = LoggerFactory.getLogger(LowMemoryDetector.class);
+
   @FunctionalInterface
-  public static interface Action {
+  public interface Action {
     void execute();
   }
```

Comment:
For what it's worth, it seems like Runnable would suffice here. No need for a new interface.